src/postprocess/chi0.jl

```diff
@@ -3,12 +3,12 @@ using IterativeSolvers
 using ForwardDiff
 using ProgressMeter
 
+# make ldiv! act as a given function
 struct FunctionPreconditioner
-    f_ldiv!
+    f! # f!(y, x) applies f to x and puts it into y
```

Comment:
Space before `#`. Also I don't like `f!`. It's too easy to overlook alltogether. how about `precondition!` or `apply!` or something like that?

Reply:
how do you mean it's easy to overlook? It's literally the only thing in this struct. I can make it `fun` if you want

Reply:
Yes. I meant if it is used or set in the code. I'm ok with `fun!`.